src/quick-tour/attract-customers.md

```diff
@@ -4,18 +4,17 @@ title: Attract New Customers
 
 Magento is packed with features that make it easy to create a “search engine friendly” websites and increase the likelihood of bringing the right customers to your site.
 
-[Search Engine Optimization]({{ site.baseurl }}{% link marketing/seo-search.md %})
-:  Magento offers powerful, native capabilities to streamline Search Engine Optimization (SEO) practices for content and site exposure that are integrated with the Admin, and tied directly into the user experience.
+Search Engine Optimization
```

Comment:
I would make these headings, or all of the content in a markdown table. This formatting is a little strange.